cuda_bindings/cuda/bindings/_path_finder/supported_libs.py

```diff
@@ -0,0 +1,386 @@
+    ),
+    "nppicc": (
+        "nppicc64_11.dll",
+        "nppicc64_12.dll",
+    ),
+    "nppidei": (
+        "nppidei64_11.dll",
+        "nppidei64_12.dll",
+    ),
+    "nppif": (
+        "nppif64_11.dll",
+        "nppif64_12.dll",
+    ),
+    "nppig": (
+        "nppig64_11.dll",
+        "nppig64_12.dll",
+    ),
+    "nppim": (
+        "nppim64_11.dll",
+        "nppim64_12.dll",
+    ),
+    "nppist": (
+        "nppist64_11.dll",
+        "nppist64_12.dll",
+    ),
+    "nppisu": (
+        "nppisu64_11.dll",
+        "nppisu64_12.dll",
+    ),
+    "nppitc": (
+        "nppitc64_11.dll",
+        "nppitc64_12.dll",
+    ),
+    "npps": (
+        "npps64_11.dll",
+        "npps64_12.dll",
+    ),
+    "nvJitLink": ("nvJitLink_120_0.dll",),
+    "nvblas": (
+        "nvblas64_11.dll",
+        "nvblas64_12.dll",
+    ),
+    "nvfatbin": ("nvfatbin_120_0.dll",),
+    "nvjpeg": (
+        "nvjpeg64_11.dll",
+        "nvjpeg64_12.dll",
+    ),
+    "nvrtc": (
+        "nvrtc64_110_0.dll",
+        "nvrtc64_111_0.dll",
+        "nvrtc64_112_0.dll",
+        "nvrtc64_120_0.dll",
+    ),
+    "nvvm": (
+        "nvvm32.dll",
+        "nvvm64.dll",
+        "nvvm64_33_0.dll",
+        "nvvm64_40_0.dll",
+    ),
+}
+
+LIBNAMES_REQUIRING_OS_ADD_DLL_DIRECTORY = (
+    "cufft",
+    "nvrtc",
+)
+
+
+def is_suppressed_dll_file(path_basename: str) -> bool:
+    if path_basename.startswith("nvrtc"):
+        # nvidia_cuda_nvrtc_cu12-12.8.93-py3-none-win_amd64.whl:
+        #     nvidia\cuda_nvrtc\bin\
+        #         nvrtc-builtins64_128.dll
+        #         nvrtc64_120_0.alt.dll
+        #         nvrtc64_120_0.dll
+        return path_basename.endswith(".alt.dll") or "-builtins" in path_basename
+    return False
+
+
+# Based on nm output for Linux x86_64 /usr/local/cuda (12.8.1)
+EXPECTED_LIB_SYMBOLS = {
+    "nvJitLink": (
+        "__nvJitLinkCreate_12_0",  # 12.0 through 12.8 (at least)
+        "nvJitLinkVersion",  # 12.3 and up
+    ),
+    "nvrtc": ("nvrtcVersion",),
+    "nvvm": ("nvvmVersion",),
+    "cudart": ("cudaRuntimeGetVersion",),
+    "nvfatbin": ("nvFatbinVersion",),
+    "cublas": ("cublasGetVersion",),
+    "cublasLt": ("cublasLtGetVersion",),
+    "cufft": ("cufftGetVersion",),
+    "cufftw": ("fftwf_malloc",),
+    "curand": ("curandGetVersion",),
+    "cusolver": ("cusolverGetVersion",),
+    "cusolverMg": ("cusolverMgCreate",),
+    "cusparse": ("cusparseGetVersion",),
+    "nppc": ("nppGetLibVersion",),
+    "nppial": ("nppiAdd_32f_C1R",),
+    "nppicc": ("nppiColorToGray_8u_C3C1R",),
+    "nppidei": ("nppiCopy_8u_C1R",),
+    "nppif": ("nppiFilterSobelHorizBorder_8u_C1R",),
+    "nppig": ("nppiResize_8u_C1R",),
+    "nppim": ("nppiErode_8u_C1R",),
+    "nppist": ("nppiMean_8u_C1R",),
+    "nppisu": ("nppiFree",),
+    "nppitc": ("nppiThreshold_8u_C1R",),
+    "npps": ("nppsAdd_32f",),
+    "nvblas": ("dgemm",),
+    "cufile": ("cuFileGetVersion",),
+    # "cufile_rdma": ("rdma_buffer_reg",),
```

Comment:
dead code?

Reply:
I added a comment to explain: commit 4d95eb4dc49e47cc79025fd6dfaedcb5d558a25c